docs/docs/cmd/planner/plan/plan-remove.md

```diff
@@ -0,0 +1,48 @@
+## planner plan remove
+
+Removes the Microsoft Planner plan
+
+## Usage
+
+```sh
+m365 planner plan remove [options]
+```
+
+## Options
+
+`i, --id [id]`
+: ID of the plan to remove. Specify either `id` or `title` but not both.
+
+`-t, --title [title]`
+: Title of the plan to remove. Specify either `id` or `title` but not both.
+
+`--ownerGroupId [ownerGroupId]`
+: ID of the Group that owns the plan. Specify either `ownerGroupId` or `ownerGroupName` when using `title`.
+
+`--ownerGroupName [ownerGroupName]`
+: Name of the Group that owns the plan. Specify either `ownerGroupId` or `ownerGroupName` when using `title`.
```

Comment:
can we specify bot `ownerGroupId` and `ownerGroupName` 🤔? in docs seems like we may

Reply:
I know normally we say `but not both`. But if you check all other commands like the Planner commands, this is only used when the option is not bound to another option.
For example this command https://pnp.github.io/cli-microsoft365/cmd/planner/bucket/bucket-list. You can see 'but not both' is used for the first 2 options which are not dependent. But other options are formulated like I did here. This applies to all other Planner commands as well.

Maybe we should discuss how we should formulate it for all docs, but for me it's clear this way. For me 'x or y' means either the first or the second. If they can be used both it should be 'x and/or y'. But that's of course how I see it.

Reply:
ok, since we already have this approach in other places lets be consistent here as well 👍